spec.md

```diff
@@ -4220,9 +4282,11 @@ decl_extmodule =
 
 decl_layer =
   "layer" , id , string , ":" , [ info ] , newline , indent ,
-    { decl_layer , newline } ,
```

Comment:
Is this intentional? Or did this clip another branch?

Reply:
woops merge artifact probably